PXB-3104 Release notes for Percona XtraBackup 8.0.34-29

alina-derkach-oaza opened the pull request:
new file:   docs/release-notes/8.0/8.0.34-29.0.md
	modified:   docs/release-notes/8.0/8.0.34-29.0.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

alina-derkach-oaza committed:
PXB-3104 Release notes for Percona XtraBackup 8.0.34-29
new file:   docs/release-notes/8.0/8.0.34-29.0.md

patrickbirch commented on docs/release-notes/8.0/8.0.34-29.0.md:
```diff
@@ -0,0 +1,29 @@
+
+This release merges the MySQL 8.0.34 code base. Percona has implemented a two-stage release process for each version. The first release primarily ensures compatibility with the latest MySQL version, to help those customers who need an updated version of Percona XtraBackup as soon as possible. The second release contains additional bug fixes and any improvements or new features.
+
+The [ZSTD compression method]() is moved to [General Availability](..//..//glossary.md#general-availability-ga) and becames default compression method for `--compress` option.
```
the default compression method for the `--compress` option.

alina-derkach-oaza replied:
done

patrickbirch commented on docs/release-notes/8.0/8.0.34-29.0.md:
```diff
@@ -0,0 +1,29 @@
+
+## Bug fix
+
+* [PXB-3073](https://jira.percona.com/browse/PXB-3073): The `xbcloud` got stuck while uploading the backup with 1 million tables.
```
An unresponsive host may cause `xbcloud` to become stuck. Added a `--timeout` option to `xbcloud`

alina-derkach-oaza replied:
done

patrickbirch reviewed (approved):
Made a few changes to wording

patrickbirch commented on docs/release-notes/8.0/8.0.34-29.0.md:
```diff
@@ -0,0 +1,29 @@
+
+## Improvements
+
+* [PXB-3007](https://jira.percona.com/browse/PXB-3007): The [ZSTD compression method]() is moved to [General Availability](..//..//glossary.md#general-availability-ga) and becomes the default compression method for the `--compress` option.
```
add a link to the compress option
create-compressed-backup.md

alina-derkach-oaza replied:
done

altmannmarcelo commented on docs/release-notes/8.0/8.0.34-29.0.md:
```diff
@@ -0,0 +1,31 @@
+
+This release merges the MySQL 8.0.34 code base. Percona has implemented a two-stage release process for each version. The first release primarily ensures compatibility with the latest MySQL version, to help those customers who need an updated version of Percona XtraBackup as soon as possible. The second release contains additional bug fixes and any improvements or new features.
+
+The [ZSTD compression method](..//..//create-compressed-backup.md) is moved to [General Availability](..//..//glossary.md#general-availability-ga) and becomes the default compression method for the [`--compress`](..//..//create-compressed-backup.md) option.
```
Add a heading `### Compression`
Keep the note about ZSTD moving to default
Add a note that quicklz has been removed from --backup but still an option to --decompress older backups.

alina-derkach-oaza replied:
updated

altmannmarcelo reviewed (changes requested):
We added a few new tickets to this release:

https://jira.percona.com/browse/PXB-3088
https://jira.percona.com/browse/PXB-3079
https://jira.percona.com/browse/PXB-2860

alina-derkach-oaza commented:
> We added a few new tickets to this release:
> 
> https://jira.percona.com/browse/PXB-3088 https://jira.percona.com/browse/PXB-3079 https://jira.percona.com/browse/PXB-2860

Added
